lib/views/pages/profile_page.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+class ProfilePage extends StatelessWidget {
```

Comment:
We don't need this one anymore because we have a profile page added by another contributor.
You can deal with it and add it in the navbar